fiorella/src/styles/layout/slider.scss

```diff
@@ -0,0 +1,153 @@
+.slide {
+  display: none;
+}
+
+.slider-container {
+  position: relative;
+}
+
+.slider-navigator {
+  display: flex;
+  justify-content: center;
+  position: absolute;
+  top: 10px;
+  width: 100%;
+}
+
+.slider-nav-button {
+  background-color: $color-white-solid;
+  border-radius: 50%;
+  border-style: none;
+  font-family: sans-serif;
+  margin: 0 5px;
+  height: 12px;
+  width: 5px;
+}
+
+.slider-nav-button-selected {
+  background-color: $color-featured;
+}
+
+.show-slide {
+  display: flex;
+}
+
+.slide {
+  flex-direction: column;
+  position: relative;
+
+  img {
+    width: 100%;
+  }
+}
+
+.slide-description {
+  bottom: 60px;
+  font-family: $font-slider-description;
+  font-style: italic;
+  font-size: 15px;
+  position: absolute;
+  right: 15px;
+  text-align: right;
+  width: 150px;
+}
+
+.slide-button {
+  display: inline;
+  background-color: transparent;
+  border-style: solid;
+  border-color: $color-featured;
+  bottom: 20px;
+  font-family: $font-slider-button;
+  font-weight: $font-weight-bold;
+  right: 15px;
+  color: $color-featured;
+  padding: 10px 30px;
+  position: absolute;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+}
+
+@media #{$tablet-view} {
+  .slider-navigator {
+    bottom: 20px;
+    display: flex;
+    justify-content: center;
+    position: absolute;
+    top: auto;
+    width: 100%;
+  }
+
+  .slide-button {
+    bottom: 40px;
+    padding: 10px 40px;
```

Comment:
Properties should be ordered bottom, font-size, padding

Reply:
Please check magmahount's comments